### PR TITLE
feat UserProfile

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added UserProfile.

### CODE REVIEW

File: Profile/Profile.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
+}
+.profile h4{
+    padding:0 1rem;
+    text-transform: uppercase;
+}
+.desp{
```
Comment (mimansha-swarup) [inline Profile/Profile.css:21]: give more meaningful class name
do it later

**Review by mimansha-swarup — APPROVED** [review]
Good to go 👍

**Review by Amandeep-sinha — APPROVED** [review]

File: Profile/Profile.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply Profile/Profile.css:21]: okay

File: Profile/Profile.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <li ><a href="/Products/Products.html">Products</a></li>
+            <li><a href="/Cart/Cart.html"><i class="fas fa-shopping-cart"><div class="items__no">5</div></i></a></li>
+            <li ><a href="/Wishlist/Wishlist.html"><i class="fas fa-heart "><div class="items__no">2</div></i> </a></li>
+            <li ><a href="/Profile/Profile.html"><i class="fas fa-user"></i></a></li>
+           
+            
+            </ul>
+        </div>
+        </header>
+        <div class="profile__box">
```
Comment (KaustubhM1997) [inline Profile/Profile.html:30]: It would be nice to use input elements here instead of div. The rest looks good!

**Pull request merged**